zstacklib/zstacklib/utils/qcow2.py

```
@@ -19,8 +19,7 @@ def __init__(self, dst_path, task_spec):
             self.dst_path = dst_path
             self.__dict__.update(daemonargs)
 
-        def __exit__(self, exc_type, exc_val, exc_tb):
-            super(ConvertTaskDaemon, self).__exit__(exc_type, exc_val, exc_tb)
+        def _exit(self, exc_type, exc_val, exc_tb):
```

Ruff (0.14.8)

Unused method argument: `exc_type`

(ARG002)